example_custom_unpacking_test.go

```diff
@@ -84,7 +85,9 @@ func Example_customUnpacking() {
 		User:          "test",
 		Pass:          "test",
 	}
-	conn, err := tarantool.Connect(server, opts)
+	ctx, cancel := context.WithTimeout(context.Background(), opts.Reconnect/2)
```

Comment:
Why not 500ms as in other tests?

Reply:
In the previous implementation we used `opts.Reconnect/2` as timeout for `dial`. So now I thought it would be correct to use that value, instead of default hardcode (even tho they are equal). Another issue that I didn't put `Connect` inside a loop for `MaxReconnects` iterations (to match the previous test). 

Maybe I should use 500ms as `context.Timeout`, but also put `Connect` inside a loop for `MaxReconnects` iterations with a sleep for `Reconnect` seconds between them. And do so for every such test.

Reply:
The example is about unpacking, not about `Connect`. So the connection code could be easy.